packages/react-composites/tests/browser/call/hermetic/ParticipantPane.test.ts

```diff
@@ -185,4 +191,114 @@ test.describe('Participant pane tests', async () => {
+    page,
+    serverUrl
+  }, testInfo) => {
+    test.skip(isTestProfileDesktop(testInfo));
+    const idleRemoteParticipant = defaultMockRemotePSTNParticipant('15556781234');
+    idleRemoteParticipant.state = 'Idle';
+
+    const initialState = defaultMockCallAdapterState([idleRemoteParticipant]);
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState));
+
+    await waitForSelector(page, dataUiId(IDS.moreButton));
+    await pageClick(page, dataUiId(IDS.moreButton));
+
+    await waitForSelector(page, dataUiId('call-composite-more-menu-people-button'));
+    await pageClick(page, dataUiId('call-composite-more-menu-people-button'));
+
+    expect(await stableScreenshot(page)).toMatchSnapshot('pstn-participant-list-idle-participant-mobile.png');
+
+    idleRemoteParticipant.state = 'Connecting';
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState));
+
+    await waitForSelector(page, dataUiId(IDS.moreButton));
+    await pageClick(page, dataUiId(IDS.moreButton));
+
+    await waitForSelector(page, dataUiId('call-composite-more-menu-people-button'));
+    await pageClick(page, dataUiId('call-composite-more-menu-people-button'));
+
+    expect(await stableScreenshot(page)).toMatchSnapshot('pstn-participant-list-connecting-participant-mobile.png');
+  });
+
+  /* @conditional-compile-remove(one-to-n-calling) @conditional-compile-remove(PSTN-calls) */
+  test('Participant should be hidden when idle to connecting mobile', async ({ page, serverUrl }, testInfo) => {
+    test.skip(isTestProfileDesktop(testInfo));
+    const idleRemoteParticipant = defaultMockRemoteParticipant('Joni Solberg');
+    idleRemoteParticipant.state = 'Idle';
+
+    const initialState = defaultMockCallAdapterState([idleRemoteParticipant]);
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState));
+
+    await waitForSelector(page, dataUiId(IDS.moreButton));
+    await pageClick(page, dataUiId(IDS.moreButton));
+
+    await waitForSelector(page, dataUiId('call-composite-more-menu-people-button'));
+    await pageClick(page, dataUiId('call-composite-more-menu-people-button'));
+
+    expect(await stableScreenshot(page)).toMatchSnapshot('participant-list-idle-participant-mobile.png');
+
+    idleRemoteParticipant.state = 'Connecting';
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState));
+
+    await waitForSelector(page, dataUiId(IDS.moreButton));
+    await pageClick(page, dataUiId(IDS.moreButton));
+
+    await waitForSelector(page, dataUiId('call-composite-more-menu-people-button'));
+    await pageClick(page, dataUiId('call-composite-more-menu-people-button'));
+
+    expect(await stableScreenshot(page)).toMatchSnapshot('participant-list-connecting-participant-mobile.png');
+  });
+
+  /* @conditional-compile-remove(one-to-n-calling) @conditional-compile-remove(PSTN-calls) */
+  test('Participant should be hidden when idle to connecting desktop', async ({ page, serverUrl }, testInfo) => {
```

Comment:
can we avoid having a mobile test and a desktop test?
It's hard to tell but looking between them it seems the only difference is the how the people pane is opened. It would be better to have a function `await openPeoplePane(isMobile)` and just have one test

Reply:
I don't think we can. problem with the `await waitforSelector` and `await pageClick` calls if the selector it is trying to use is not present the function will timeout the test. moving this to another function I don't think will solve this problem.